src/libs/Navigation/AppNavigator/ModalStackNavigators.js

```diff
@@ -468,7 +468,7 @@ const SettingsModalStackNavigator = createModalStackNavigator([
             return ReimbursementAccountPage;
         },
         name: 'ReimbursementAccount',
-        initialParams: {stepToOpen: CONST.BANK_ACCOUNT.STEP.BANK_ACCOUNT},
+        initialParams: {stepToOpen: ''},
```

Comment:
This was making us navigate first to `/bank-account/BankAccountStep` which is an incorrect url, it should have been `/bank-account/` or `/bank-account/new` as this what this function is expecting:

https://github.com/Expensify/App/blob/332d464829c394a9ef8606c29da228287bf6ea12/src/pages/ReimbursementAccount/ReimbursementAccountPage.js#L149-L166